src/lib/bootloader/grub2base.rb

```diff
@@ -361,7 +361,7 @@ def propose_terminal
     def propose_timeout
       return if grub_default.timeout
 
-      grub_default.timeout = "8"
+      grub_default.timeout = Yast::ProductFeatures.GetIntegerFeature("globals", "boot_timeout").to_s
```

Comment:
So, what is default if not defined?

Reply:
It is always defined. See - https://github.com/yast/yast-yast2/pull/1240/files#diff-e3d91269dec42511487bf9598e632fe2da743b05435e0cf3c77da23e520c0f77R55

Reply:
.. so, maybe the dependency should be updated requiring that specific version? https://github.com/yast/yast-bootloader/blob/0e1d4068e86aaf04a602a62ee30a4a72395aa1c2/package/yast2-bootloader.spec#L46

Reply:
https://github.com/yast/yast-bootloader/blob/0e1d4068e86aaf04a602a62ee30a4a72395aa1c2/package/yast2-bootloader.spec#L29